src/main/java/seedu/address/model/PropertyBook.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.property.Property;
+import seedu.address.model.property.UniquePropertiesList;
+
+/**
+ * Wraps all data at the address-book level
+ * Duplicates are not allowed (by .isSameProperty comparison)
+ */
+public class PropertyBook implements ReadOnlyPropertyBook {
+    private final UniquePropertiesList properties;
+
+    {
```

Comment:
just wondering why there are open brackets at line 18 and 20?

Reply:
This is to replicate the AddressBook.java into the propertyBook.java as much as possible. The purpose of the open brackets as commented in AdressBook.java is:
`
/*
     * The 'unusual' code block below is a non-static initialization block, sometimes used to avoid duplication
     * between constructors. See https://docs.oracle.com/javase/tutorial/java/javaOO/initial.html
     *
     * Note that non-static init blocks are not recommended to use. There are other ways to avoid duplication
     *   among constructors.
     */
`
I'll add this comment to the PropertyBook.java as well